Database/Corrections/sodQuestFixes.lua

```diff
@@ -2576,6 +2576,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -416049,
             [questKeys.zoneOrSort] = sortKeys.DRUID,
         },
+        [90165] = {
+            [questKeys.name] = "Twisted Faith",
+            [questKeys.startedBy] = {{3268,3267,3266,3265,3457},nil,{208765}},
```

Comment:
No need to add the item starters. They are not used.
```suggestion
            [questKeys.startedBy] = {{3268,3267,3266,3265,3457}},
```